ansible/group_vars/all/piecewise_global_config.yml

```diff
@@ -0,0 +1,278 @@
+
+########
+# DEPLOYMENT FILE LOCATIONS
+########
+
+# These paths specify where specific components will be deployed. 
+# These can be largely unchanged, unless your system requires different paths.
+
+# Where the base files from your repo will be copied on the remote server.
+base_path: /opt
+
+# Temporary location where the Git repo will be cloned and the build scripts
+# will be run before going live.
+temp_path: "{{ base_path }}/tmp"
+
+# Where the production site symlink will exist.
+site_path: "{{ base_path }}/piecewise_web"
+
+########
+# MAPPING & AGGREGATION CUSTOMIZATIONS
+########
+
+###
+# Geodata and Database Information
+#
+# The section below defines variables used to import your geodata into the database.
+
+# Save the geofiles you intend to use for your Piecewise aggregations of
+# M-Lab data in the folder: /local_customizations/geofiles/
+
+# Add the name of the shapefile file to be used.
+shape_file: CENSUS10_blkgrp_WGS.shp
+
+# The variables below are used when importing your shapefile into postgres.
+#
+# 'dbKey' should be the field name of the unique key in your shape file.
+# 'dbKeyDataType' should be the data type of dbKey
+# 'shape_projection' specifies the SRS reprojection/transform to use for this shapefile.
+# 'geometry_type' specifies the geometry type for this layer
+# 'layer_name' is the name you wish to be used for this shapefile layer in postgres
+dbKey: geoid10
+dbKeyDataType: string
+shape_projection: "EPSG:4326"
+geometry_type: MultiPolygon
+layer_name: seattle_blkgrpce10
+
+# The variables below define the aggregation name and table name for aggregate statistics
+aggregation_name: "by_census_block"
+aggregation_stats_table_name: "block_statistics"
+
+# Define the latitude and longitude for the southwest and northeast corners of a bounding 
+# box, defining the region where you are interested in pulling M-Lab NDT data.
+map_coordinate_southwest: 
+  lat: 47.3630134401
+  lon: -122.6733398438
+
+map_coordinate_northeast: 
+  lat: 47.8076208172
+  lon: -121.9509887695
+
+# The four variable below are derived from your bounding box coordinates above.
+map_bounding_box: "{{ map_coordinate_southwest.lon }},{{ map_coordinate_southwest.lat }},{{ map_coordinate_northeast.lon }},{{ map_coordinate_northeast.lat }}"
+
+# The two dates below define the date range for M-Lab NDT test data. In production,
+# it is advisable to set 'map_data_data_end_date' to a date well in the future, for
+# example: "Dec 31 2050 00:00:00"
+map_data_start_date: "Jan 1 2017 00:00:00"
+map_data_end_date: "Feb 15 2017 00:00:00"
+
+###
+# WEBSITE MAP INFORMATION
+#
+# The section below defines variables to be used on the website map.
+
+# Piecewise requires a topojson file for the website to use. This topojson file must
+# be created from the shapefile you defined above. 
+# Save your topojson file into the folder /local_customizations/geofiles
+# and then enter its filename here.
+topojson_file: seattle_census10_blockgroups.topojson
```

Comment:
this might be out of scope at the moment and is more of a @dwins question, but can't we generate the topojson file in postgres instead of needed the shp & topojson?